packages/gatsby-theme-carbon/src/components/Code/Code.module.scss

```diff
@@ -11,13 +11,17 @@
     position: relative;
     display: inline;
     padding: 0 0.5em;
-    font-size: 0.75em;
+    font-size: 0.875em;
```

Comment:
```suggestion
    font-size: 0.75rem;
```

Reply:
looks like it shoulda been `rem` so it's relative to the root instead of whatever the current font-size is.

Reply:
@vpicone changing it to `.75rem` makes it too small(12). should be 14px for body text.

Reply:
Whoops! My bad, I’d just leave the r in there so it’s /16 instead of /whatever the current font size is.